#Позиция ключа, который хотим переключить
int8 id

# Новое значение ключа
int8 state